language-server/modules/langserver-core/src/test/resources/completion/field_access_expression_context/config/field_access_ctx_config28.json

```diff
@@ -604,6 +604,25 @@
         "title": "editor.action.triggerParameterHints",
         "command": "editor.action.triggerParameterHints"
       }
+    },
+    {
```

Comment:
@nadeeshaan, @malinthar, can y'all please review these changes? They LGTM.

Reply:
Got them verified by @TharushiJay

Reply:
Thanks. Will go ahead and merge!